docs/add_test.md

```diff
@@ -124,9 +124,9 @@ Tests should:
 - Include the test name and a description of the test in the report output
 - Include an informative line in the summary table
 
-Integration tests don't need to be tedious and, if you're developing one test and seeing a consistent failure on Travis CI, isolate your problem and run _just that part_ of the integration test both locally and on Travis CI.
+Integration tests don't need to be tedious and, if you're developing one test and seeing a consistent failure on [Github actions](https://github.com/faucetsdn/daq/actions), isolate your problem and run _just that part_ of the integration test both locally and on [Github actions](https://github.com/faucetsdn/daq/actions).
```

Comment:
nit: in my opinion it's probably overkill to link all of them since they're all the same. Just like when a new term is introduced using _italics_, it's only done the first time.